modules/garbage.py

```diff
@@ -2,124 +2,108 @@
 
 class GC(object):
 
-    def __init__(self, env, heap, threshold=0.9, sleep=2):
+    def __init__(self, env, server, threshold=0.9, sleep=0.02):
         self.env = env
-        self.heap = heap
+        self.server = server
+        self.heap = server.heap
         self.threshold = threshold
         self.sleep = sleep
-        self.collect_exe = None
+        self.collect_exe = False
```

Comment:
What does this mean?

Reply:
As before, the name isn't expressive. That collect_exe is a boolean variable used to know if the GC is collecting the trash. The objective of that, is to avoid starting two collects at same time.

Reply:
The name was changed to "collecting_trash", what you think @danielfireman  and @dfquaresma ?

Reply:
Sounds good to me. Would be too long have that variable named as is_collecting_trash?

Reply:
What do you think of `Is_gcing`?

Reply:
Fits better. We'll update it.